lint/linter/test-descriptions.ts

```diff
@@ -116,6 +116,15 @@ export const processData = (
     processApiData(data, path, errors);
   }
 
+  if (data.description === `<code>${path.split('.').at(-1)}</code>`) {
```

Comment:
Like in https://github.com/mdn/browser-compat-data/pull/21886, `description` can be an object because we have data like "api.GPUAdapterInfo.description", so I think we need to avoid getting trapped into that.

Reply:
We're already all set there; the `data` variable is always the `__compat` key of a BCD entry in this linter!